src/state/hass-element.ts

```diff
@@ -26,6 +27,7 @@ export class HassElement extends ext(HassBaseEl, [
   DisconnectToastMixin,
   connectionMixin,
   NotificationMixin,
+  QuickOpenMixin,
```

Comment:
This should not be part of the HassElement but of HomeAssistant element. This element is reused in different places too.

Reply:
Is that `HomeAssistantAppEl`? `HomeAssistantMain`? Other? I'm not seeing an element named explicitly "HomeAssistant".

I went ahead and tried on HomeAssistantMain. I used the naive approach of duplicating how QuickOpenMixin is added to HassElement, but now chunk loading fails:
![image](https://user-images.githubusercontent.com/1480827/95113467-5c4a6400-06f7-11eb-8595-80918512177e.png)

Reply:
`HomeAssistantAppEl` === `<home-assistant>`

Reply:
Ha. I've been banging my head against a wall trying to figure out why doing this giving me chunk loading errors in console:

`export class HomeAssistantAppEl extends QuickOpenMixin(HassElement) {`

Turns out I just had to kill the `script/develop` watcher process and restart. Works like a charm now.

Reply:
Done and pushed